src/module-1/styles/css/style.css

```diff
@@ -0,0 +1,156 @@
+*,
```

Comment:
consider to split your css into separate files, like `global.css`, `reset.css` (if needed), `list.css`